packages/legacy-workbench/src/js/angular/guides/tour-lib-services/shepherd.service.js

```diff
@@ -986,7 +987,10 @@ function ShepherdService($location, $translate, LocalStorageAdapter, LSKeys, $ro
         typeIcon.setAttribute('gdb-tooltip', $translate.instant('guide.step-type.' + iconTooltip));
         typeIcon.setAttribute('tooltip-placement', 'bottom-right');
 
-        angular.element(currentStepElement.querySelector('.shepherd-header')).prepend(typeIcon);
+        const header = currentStepElement.querySelector('.shepherd-header');
```

Comment:
I can see that the css class used in this selector has a typo and that you haven't changed it. Can you confirm that this expression is properly doing what's expected?

Reply:
The class "shepherd-header" is part of the native ShepherdJS library and is applied to the header of the dialog.

Reply:
I see. It's my bad. :)